activemodel/lib/active_model/dirty.rb

```diff
@@ -139,6 +139,11 @@ def initialize_dup(other) # :nodoc:
       @mutations_from_database = nil
     end
 
+    def as_json(options = {})
```

Comment:
Documentation is missing!!

Reply:
Added Documentation, please review again.